client/config.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import "time"
+
+// Config represents the configuration parameters for state channel client.
+type Config struct {
+	// Addresses of on-chain contracts used for establishing state channel network.
+	Adjudicator string
+	Asset       string
+
+	// URL of the blockchain node.
+	ChainURL string
+	// ChainConnTimeout is the timeout used when dialing for new connections to the on-chain node.
+	ChainConnTimeout time.Duration
+
+	// Path to directory containing persistence database
+	PersistenceDir string
```

Comment:
Is it better to use DatabaseDir or something related to DB and Dir, like PersistenceDbDir  instead of PersistenceDir, so that from the name itself we can catch it is a database directory path.?

Reply:
I have updated the variable name to `DatabaseDir`.